example/src/pages/index.tsx

```diff
@@ -70,12 +77,33 @@ const IndexPage = () => {
     };
   }, [searchParams]);
 
+  /**
+   * Retrieve init fee surrency example
+   */
+  const getInitFeeCurrency = useCallback(async () => {
```

Comment:
Is it the purpose of this function to not return anything nor change the state of the app?

Reply:
Yes, It is only a example to Moonpay, as they were asking receive this value as a query attribute